### ISSUE TITLE
Add documentation about eBPF and BTF support in different kernels

### ISSUE THREAD

**Issue by v-thakkar** [parca-dev/parca-agent#333]
eBPF was introduced in one of the 3.x kernel and BTF support was added in one of the 5.x version. Although a lot of these features have been backported in different mainline stable kernels as well as in distro-specific kernels. We should add documentation with some commands so that it's easier for users to identify if their host kernel has BPF and BTF support or not.

**Comment by brancz** [comment]
Could we maybe even have the Parca Agent do a few tests on startup to identify whether the features it needs are available and error out with that exact feature if it isn't available?

**Comment by brancz** [comment]
Agreed, definitely, both are important and useful!

**Comment by kakkoyun** [comment]
Yep, sounds good. If you want to handle that, please go ahead. Otherwise, I can handle it.

**Comment by v-thakkar** [comment]
Sure, I can take care of it.
